src/Broker/Broker.c

```diff
@@ -0,0 +1,74 @@
+#include "Broker.h"
+
+#include <string.h>
+#include <stdint.h>
+
+static Packetizer packetizer_data = { 0 };
+static uint8_t packetizer_buffer[BROKER_BUFFER_SIZE];
+
+extern driver_send_function bus_to_driver_send_function[SYSTEM_BUSES_NUMBER];
+extern void* bus_to_driver_private_data[SYSTEM_BUSES_NUMBER];
+extern deliver_function interface_to_deliver_function[INTERFACE_MAX_ID];
+
+extern void Broker_acquire_lock();
+extern void Broker_release_lock();
+
+void
+Broker_initialize()
+{
+    Packetizer_init(&packetizer_data);
+}
+
+void
+Broker_deliver_request(enum RemoteInterface interface, uint8_t* data, size_t length)
```

Comment:
const args

Reply:
done

Reply:
what about the enum?

Reply:
Ou, I've added it only in header. Fixed.